tests/everflow/test_everflow_ipv6.py

```diff
@@ -9,7 +9,7 @@
 from everflow_test_utilities import setup_info  # noqa: F401, E501 lgtm[py/unused-import] pylint: disable=import-error
 
 pytestmark = [
-    pytest.mark.topology("t0", "t1", "m0", "mx")
```

Comment:
Are you plan to support mx in another PR?

Reply:
Yes